go/private/context.bzl

```
@@ -884,6 +887,10 @@ go_config = rule(
             providers = [BuildSettingInfo],
         ),
         "amd64": attr.string(),
+        "pgoprofile": attr.label(
+            mandatory = True,
+            allow_files = True,
```

```suggestion
            allow_single_file = True,
```
is more restrictive and probably what we want here (assuming the empty `filegroup` trick still works with it).

---

Even if it doesn't, do we expect all profile to have a `pprof` file extension? If so, we could restrict this to such files.